pkg/controller/petset/pet.go

```diff
@@ -297,22 +297,23 @@ type petHealthChecker interface {
 // It doesn't update, probe or get the pod.
 type defaultPetHealthChecker struct{}
 
-// isHealthy returns true if the pod is running and has the
-// "pod.alpha.kubernetes.io/initialized" set to "true".
+// isHealthy returns true if the pod is running and if it has the
```

Comment:
nit: pod is running and ready, and please reword to "if the pod is running and ready. If it has the ... set to false, pod state is ignored."